azure-pipelines/simple_test_framework.yml

```diff
@@ -29,6 +29,10 @@ jobs:
       maxParallel: 4
 
     steps:
+      - bash: |
+          echo "##vso[task.setvariable variable=shellopts]errexit"
+        displayName: "Force exit on error (bash) "
+
```

Comment:
@epassaro Is this correct?

Reply:
Yes, but we must try it. For example, a new bash step including:

```
touch -j
echo "hello world"
```

... should fail.

Reply:
Can you explain how to do this? Because the jobs don't run if the previous job fails. So how to use this variable?

Reply:
It's only a test. I want to make it fail on purpuse. 

With the normal Azure Pipelines behavior a step like

```bash: |
            touch -j
            echo "hello world"
```

Returns a green check and keep running the pipeline. That behavior is not desired because we want to know if something is failing (and stop the pipeline). 

If the `#vso...` piece of code works as we want, then the pipeline should fail if any bash command fails. But we don't know, nobody made this before.

See: https://github.com/microsoft/azure-pipelines-agent/issues/1803

Reply:
@epassaro would it be a good idea to add each bash command as a step in the `steps:`. This way we can describe that each step is dependent on the previous steps and then when any of the steps fail, the pipeline fails right away.